src/StackAdmin/Azs.AzureBridge.Admin/readme.md

```diff
@@ -57,16 +57,52 @@ csproj: Azs.AzureBridge.Admin.csproj
 psd1: Azs.AzureBridge.Admin.psd1
 psm1: Azs.AzureBridge.Admin.psm1
 
-directive:
+directive:  
+  # Add AzureBridge prefix to subjects to match existing method names
   - where:
-      model-name: ActivationResource
+      subject: Activation
     set:
-      suppress-format: true
+      subject-prefix: AzureBridge
+  - where:
+      subject: DownloadedProduct
+    set:
+      subject-prefix: AzureBridge
+  - where:
+      subject: Product
+    set:
+      subject-prefix: AzureBridge
```

Comment:
you should be able to add a module level subject-prefix, @rakku-ms can you pls provide a sample?

Reply:
see my next comment - line 106